powershell/upload/README.md

```diff
@@ -0,0 +1,142 @@
+# File API PowerShell upload examples
+
+A collection of examples to upload files with the File API using **PowerShell**.
+
+## Prerequisites
+
+- The script was prepared for PowerShell version 5.1 or above. With lower versions it might not work properly.
+- Files to be uploaded cannot be bigger than 100 megabytes.
```

Comment:
For consistency, use 'MB' instead of 'megabytes' or change the download readme to use 'gigabytes' instead of 'GB'.

Reply:
I will change it